test/jdk/java/awt/dnd/ImageDecoratedDnD/ImageDecoratedDnD.java

```diff
@@ -21,177 +21,84 @@
  * questions.
  */
 
-/*
-  test %W% %E%
-  @bug 4874070
-  @summary Tests basic DnD functionality
-  @author Your Name: Alexey Utkin area=dnd
-  @run applet/manual=yesno ImageDecoratedDnD.html
-*/
-
-import java.applet.Applet;
-import java.awt.*;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Frame;
+import java.awt.Panel;
 import java.awt.dnd.DragSource;
 
+/*
+ * @test
+ * @bug 4874070
+ * @summary Tests basic DnD functionality
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual ImageDecoratedDnD
+ */
 
+public class ImageDecoratedDnD {
+    private static final String INSTRUCTIONS = """
+            When test runs a Frame which contains a yellow button labeled "Drag ME!"
+            and a RED Panel will appear.
+
+            1. Click on the button and drag to the red panel by pressing
+               the "CTRL" key on the keyboard.
+
+            2. When the mouse enters the red panel during the drag, the panel
+               should turn yellow.
+
+                On the systems that supports pictured drag, the image under the drag-cursor
+                should appear.
+                "Image under drag-cursor" is a translucent blue rectangle + red circle
```

Comment:
```suggestion
                On the systems that supports pictured drag, the image under the 
                drag-cursor should appear.
                "Image under drag-cursor" is a translucent blue rectangle + red circle
```

In the instructions window, there's an extra newline due to exceeding the character limit per line.